src/resource_monitor.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @fileoverview monitor and manage epsagon resource usage
+ */
+
+const schedule = require('node-schedule');
+const { utils } = require('epsagon');
+const traceContext = require('./trace_context');
+
+const MAX_TRACERS = parseInt(process.env.EPSAGON_RESOURCE_MAX_TRACERS || '50', 10);
```

Comment:
maybe change the default to be 0?

Reply:
@galbash what do you think?

Reply:
I would start with 50 - just cause hopefully we won't leak in most cases, worst case we can change it before sending them